spec/javascripts/components/expander-spec.js

```diff
@@ -19,62 +19,63 @@ describe('An expander module', function () {
 
   describe('in default mode', function () {
     beforeEach(function () {
-      expander = new GOVUK.Modules.Expander()
-      $element = $(html)
-      expander.start($element)
+      $element = document.createElement('div')
+      $element.innerHTML = html
+      new GOVUK.Modules.Expander($element.querySelector('.app-c-expander')).init()
```

Comment:
Not a blocker but you could have got away with a simpler change to these tests that just changed these beforeEach blocks.

E.g:

```diff
-      expander = new GOVUK.Modules.Expander()
       $element = $(html)
-      expander.start($element)
+      expander = new GOVUK.Modules.Expander($element[0])
+      expander.init()
```

I don't think you should change the code you've commited, as I think it's better with the lower jQuery usage but I can imagine it adds up some extra work in completing this task and reviewing the changes, so it shouldn't be a necessary step in each of these conversions.

Reply:
thanks! I will keep that in mind for the next module initialiser conversions.